src/components/TalentPage/TalentPage.module.scss

```diff
@@ -0,0 +1,133 @@
+.proofs {
+    width: 100%;
+    height: 250px;
```

Comment:
You should not specify a fixed `height` for this block, because the content can be larger than this `height` and then there will be problems. It's better to use the `min-height` constraint.